lib/ansible/modules/database/postgresql/postgresql_ext.py

```diff
@@ -34,6 +34,12 @@
       - name of the extension to add or remove
     required: true
     default: null
+  version:
+    description:
+      - version of the extension to add or remove (or update to)
+    required: false
```

Comment:
`false` and `null` are default values for `required` and `default` respectively, both fields could be removed.